src/software/world/world.cpp

```diff
@@ -151,12 +141,12 @@ boost::circular_buffer<Timestamp> World::getTimestampHistory()
 
 const GameState &World::gameState() const
 {
-    return current_refbox_game_state_;
+    return current_game_state_;
 }
 
 GameState &World::mutableGameState()
```

Comment:
Since we removed the other `mutable` getter functions, can this be removed as well?

Reply:
GameState is a bit of a mess, so I opened #1481 to refactor and remove mutable